app/src/main/res/values-pt-rBR

```diff
@@ -0,0 +1,401 @@
+    <string name="action_search_roms">Pesquisar ROMs</string>
+    <string name="action_sort_roms">Ordenar ROMs</string>
+    <string name="action_boot_firmware">Firmware de inicialização</string>
+    <string name="action_refresh_rom_list">Atualizar lista de ROMs</string>
+    <string name="hint_search_roms">Pesquisando ROMs…</string>
+    <string name="update_available">Atualização disponível: %1$s</string>
+    <string name="skip_update">Pular</string>
+    <string name="downloading_update">Baixando atualização…</string>
+    <string name="starting_download">iniciando download…</string>
+    <string name="download_progress_sizes">%1$.1fMB/%2$.1fMB</string>
+    <string name="update_download_failed">Falha ao baixar a atualização</string>
+
+    <string name="dsiware_manager">Gerenciador do DSiWare</string>
+    <string name="no_dsiware_titles_installed">Você não tem nenhum título DSiWare instalado</string>
+    <string name="dsiware_manager_no_dsi_setup">Você não configurou o sistema DSi</string>
+    <string name="dsiware_manager_invalid_dsi_setup">A configuração do seu sistema DSi é inválida</string>
+    <string name="import_dsiware_title">Importar título do DSiWare</string>
+    <string name="dsiware_manager_setup">Configurar</string>
+    <string name="dsiware_manager_fix_setup">Corrigir configuração</string>
+    <string name="dsiware_manager_load_error">Ocorreu um problema ao listar seus títulos DSiWare. Verifique a configuração do seu DSi</string>
+    <string name="dsiware_manager_import_title_error_open_nand_failed">Não foi possível abrir a NAND</string>
+    <string name="dsiware_manager_import_title_error_open_file_failed">Erro ao abrir o arquivo selecionado</string>
+    <string name="dsiware_manager_import_title_error_not_dsiware_title">Não é um título DSiWare</string>
+    <string name="dsiware_manager_import_title_error_title_already_imported">Título DSiWare já importado</string>
+    <string name="dsiware_manager_import_title_error_insatll_failed">Falha ao instalar o título</string>
+    <string name="dsiware_manager_import_title_error_unknown">Ocorreu um erro desconhecido</string>
+    <string name="dsiware_import_from_file">A partir do arquivo</string>
+    <string name="dsiware_import_from_rom_list">A partir da lista de ROMs</string>
+
+    <string name="select_dsiware_title">Selecionar título DSiWare</string>
+    <string name="no_dsiware_roms_found">Nenhuma ROM DSiWare encontrada</string>
+
+    <string name="info_fps">FPS: %1$d</string>
+    <string name="info_loading">CARREGANDO…</string>
+    <string name="save_state_slot">%1$s.</string>
+    <string name="failed_save_state">Falha ao salvar o estado</string>
+    <string name="failed_load_state">Falha ao carregar o estado</string>
+    <string name="cant_load_empty_slot">Não é possível carregar um slot de estado de salvamento vazio</string>
+    <string name="failed_reset_emulation">Falha para redefinir a emulação</string>
```

Comment:
> What do you think about using "reiniciar" instead of "redefinir"?

In case a: <string name="failed_reset_emulation">Falha para redefinir a emulação</string>

Wouldn't it have to be: <string name="failed_restart_emulation">Falha para reiniciar a emulação</string>

🤔